ote/src/cljs/ote/style/table.cljs

```diff
@@ -0,0 +1,9 @@
+(ns ote.style.table
+  "Table styles."
+  (:require
```

Comment:
Is this the correct place for these :requires? I don't see references to these symbols in the file.